### ISSUE TITLE
Exit Feature

### ISSUE THREAD

**Issue by MANIKTANEJA3** [GoogleForCreators/web-stories-wp#3452]
There is no exit feature available in the "Tips to make the most of web stories" - catalog
It should have some sort of escape sequence available so that the user doesn't have to go to the previous page.
Or 
There should be a redirecting button at every chapter of the catalog so that it's easier to navigate.

![Task](https://user-images.githubusercontent.com/49128676/88472039-deb2ec80-cf2c-11ea-87ac-b46c6be44bdf.jpeg)

**Comment by ernee** [comment]
@MANIKTANEJA3 thanks for the feedback! 

I believe you accessed this via the [website for the beta](https://google.github.io/web-stories-wp/beta/)? If so, [the tips](https://google.github.io/web-stories-wp/beta/tips.html) are in the [Web Stories format](https://amp.dev/about/stories/). Each page provides an individual tip and on the last page there is a link out to [more best practices](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/). 

I can share your feedback with the team about some way to be redirect back, perhaps to the Beta website again.

**Comment by ernee** [comment]
@MANIKTANEJA3 just a general tip that you can use your browser navigation's back button to return to the website after launching the story. Thanks again for sharing your feedback!

**Comment by MANIKTANEJA3** [comment]
@ernee yes but that takes up loading time; as a user I'd prefer that the stories would have an exit feature and would be more of a pop up than the current layout. 
Just a suggestion.
But I can close this issue if you'd like.

**Comment by ernee** [comment]
@MANIKTANEJA3 thanks for your suggestion!

 We actually have an enhancement for consideration that is like your request for a pop up like experience for the Stories embed block that you would place inside a WordPress post: https://github.com/google/web-stories-wp/issues/2241

If this were dismissible, would that improve embedded stories for you?

It's to be determined if this will be implemented on the Beta website (tutorial story), but feel free to follow [the issue]( https://github.com/google/web-stories-wp/issues/2241) for stories in general. 

I think it's ok to keep this issue open in case we have updates regarding the Beta website. Would be great to get your feedback!

**Comment by MANIKTANEJA3** [comment]
Sure ma'am, I'll keep up with the issue. Thanks for letting me know.

**Comment by lopesval** [comment]
Anyone that i showed this amazing plugin to either tries to swipe down or expects a little closing or dismissal cross at the top. We always expect a way back to a previous page for anything on the internet... and nobody i showed this to (with all my excitement) has thought of the back button.

**Comment by ernee** [comment]
@MANIKTANEJA3 @lopesval thanks again for feedback on this request/ sharing these use cases. I've [opened a request](https://github.com/ampproject/amphtml/issues/29949) for this in the AMP project's repository. Feel free to follow it and we'll also keep you updated on the progress here.

**Comment by MANIKTANEJA3** [comment]
@ernee thanks for the update I'll surely follow it up.

**Comment by jeanrinvil** [comment]
For the moment I use `web_stories_story_head` action with `get_post_type_archive_link` to achieve this.

Example : 
```
add_action('web_stories_story_head', "backbutton");
function backbutton()
{
    echo '
    <a href="' . get_post_type_archive_link('web-story') . '" target="_self" title="Toutes les stories"  style="position: fixed; top: 0;color: white;z-index: 9;padding: 1rem;"><svg width="25" xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" viewBox="0 0 477.862 477.862"
	style="enable-background:new 0 0 477.862 477.862;" xml:space="preserve" fill="#ffffff">
	<path d="M187.722,102.856V17.062C187.719,7.636,180.076-0.003,170.65,0c-4.834,0.001-9.44,2.053-12.676,5.644L4.375,176.311
			c-5.617,6.256-5.842,15.67-0.529,22.187l153.6,187.733c5.968,7.295,16.72,8.371,24.016,2.403c3.952-3.233,6.249-8.066,6.26-13.172
			v-85.043c134.827,4.386,218.965,62.02,256.888,175.787c2.326,6.96,8.841,11.653,16.179,11.656c0.92,0.003,1.84-0.072,2.748-0.222
			c8.256-1.347,14.319-8.479,14.319-16.845C477.855,259.818,356.87,112.174,187.722,102.856z" /></svg></a>
    ';
}

```

**Comment by swissspidy** [comment]
@jeanrinvil I really don't recommend doing that as it will break AMP validation for all your stories.

**Comment by dadarkman** [comment]
The need for a close button becomes even clearer when the Stories are displayed on a homepage, like in my case. My stories are in a carousel on the homepage of my site. So, on the desktop, after a story is seen and a user hits the Back button to exit it, the user ends up reloading the homepage. Not quite a good experience! On mobile, it's worst. Again, the Web Stories are on the homepage. Hitting the Back button on Chrome mobile forces the user to exit the site altogether and ends up on whatever previous page he/she was browsing prior to coming to my site.  
In short, this is currently a bad user experience and kind of defeats the very purpose of trying to make people stay on a site longer.

There are many other issues with Web Stories' UX/UI but I think this one should get a little bit more attention in the fix department.

Keep up the great work!

**Comment by swissspidy** [comment]
@dadarkman Thanks for your valuable feedback!

We are actively working on improved ways to embed stories (including carousels), where the story will be opened in a fullscreen lightbox with a close button, without ever leaving the page you're on. I believe this will address most concerns here.

If you want to open another issue for other UX issues you're encountering, please feel more than welcome to do so!

**Comment by LuckynaSan** [comment]
@dadarkman @jeanrinvil @MANIKTANEJA3 @lopesval Greetings! This exit feature is now available in version 1.5.0. of the Web Stories for WordPress plugin. Hopefully, you've had a chance to update the plugin and enjoy some of the new features. We will close this issue here, but feel free to [open a new support topic](https://wordpress.org/support/plugin/web-stories/#new-post) in the WordPress.org support forum should you encounter any issues. Also, if you'd like to [share your feedback](https://wordpress.org/support/plugin/web-stories/reviews/) about the plugin we'd love to hear it! Thank you.

**Comment by dadarkman** [comment]
WAIT!

Hi, 

I know that the thread was closed with the understanding that the "close" button was added to version 1.5.0. However, that's weird cause I don't have the feature yet on my end! I've been running that version for over a week now. I've made a few new Stories with it and tested them on both desktop and mobile as usual. Still, I don't have a close button (or similar icon) pointing to such. I went to settings and nothing exist there neither. So, is that a problem on my end or something else is happening?

Thanks

Mobile screenshot as of 4/2/21
<a href="https://imgur.com/9wA9WDR"><img src="https://i.imgur.com/9wA9WDR.jpg" title="source: imgur.com" /></a>

Desktop screenshot as of 4/2/21
<a href="https://imgur.com/O6Uc4J6"><img src="https://i.imgur.com/O6Uc4J6.jpg" title="source: imgur.com" /></a>

**Comment by swissspidy** [comment]
@dadarkman A single web story is just a regular web page that doesn‘t have any close button or anything. It‘s just a web page. „Closing“ it would mean closing the browser tab. There are no plans to add a „Close“ button that will just lead users to your homepage.

_However_, if you embed stories in a blog post or another area of your site, the story will open in a lightbox which does have a close button. The button closes the lightbox so users can continue reading the blog post

**Comment by dadarkman** [comment]
This is confusing and frustrating to say the least but I guess there's a been a disconnect/misunderstanding of the need of the close button?

On January 27th, you replied to my post with the following:
"We are actively working on improved ways to embed stories (including carousels), where the story will be opened in a fullscreen lightbox with a close button, without ever leaving the page you're on. I believe this will address most concerns here." 

KEYWORDS TAKEWAY: "(including carousels)"

But then on March 4th, the tone changed to:
"However, if you embed stories in a blog post or another area of your site, the story will open in a lightbox which does have a close button. The button closes the lightbox so users can continue reading the blog post"

KEYWORDS TAKEWAY: "There are no plans to add a „Close“ button that will just lead users to your homepage."

I mean, for people who choose to post Stories on a homepage or any other page that is NOT a blog page are out of luck, forever? The Stories opens and functions perfectly without it being embedded in a blog post, the team @gooogle working on Stories development is now telling me that it CAN'T and WON'T provide a close button? Is that what it is? So, currently, it is mandatory that the stories are embedded in a blog post to have the gracing of a close button? Woah man! 

**The fact is this: It's NOT about the Stories page closing back to my homepage. It is a simple basic need to have a close button, PERIOD! No matter where that Stories resides on the site. As programmers and coders of that plugin, you guys can't possibly want to make me believe that this is some hard task to accomplish**

Disappointed but it is what it is...

Thanks anyway.

**Comment by swissspidy** [comment]
> I mean, for people who choose to post Stories on a homepage or any other page that is NOT a blog page are out of luck, forever?

Not exactly. I specifically mentioned that when embedding stories _anywhere_ they will open in a lightbox that can be closed.

A standalone web story doesn‘t have a close button. Nor does this web page right here.

**Comment by dadarkman** [comment]
The more people get to adapt Stories as a way to display content on their site, outside of the blog, the demand/request/need for going back to the page without the reloading will persist. Honestly , if more users knew about this Github page, I bet my wallet that this request would increase tenfold. The reason is because as it is, browsing Stories away from the Discover page is currently NOT user friendly. I can't be the only person who sees the problem with this, can't be.

Continuing to invest time and effort in producing Stories knowing fully well that the user experience is lacking is something I have to think about at this point. I don't know, maybe I'll wait for Facebook to offer an Instagram plugin, haha!

Anyhow, thanks and wishing you all well!

**Comment by dadarkman** [comment]
OK, this just came to my attention...
All the time it was being mentioned that you've added a close button... That close button was added to Gutenberg blocks!
I never seen it because I'm using Elementor! See, that's where the disconnect is/was.

So, I guess before I throw in the towel altogether, the question now to you guys is the following: Can at some point, the Google team behind Stories develop an Elementor widget to make the same features available for Elementor users?  

Thanks

**Comment by dadarkman** [comment]
OK, I've deleted my last 2 posts because I realize they were unnecessary as I've figured the root of the misunderstanding. Actually, that was on my end. 
I'm using Elementor. So, from the beginning I've been displaying Web Stories on my site using an Elementor addon (Elementor Pack). As you guys updated the plugin with the close button, that feature somehow doesn't appear through that particular addon. So, when I received the email saying that the thread was closed, I was like what? I still don't have the close button! So, I sprinted here to reopen the thread and started this debate. While I did that, I completely forgot to check on Gutenburg or the native widget in Elementor. Sure enough, I've applied the native widget in Elementor, and to my delight, I can see the close button now on the homepage and archives ☺. So, although this particular native widget is basic compare to the Elementor addon that I'm currently using, I'm surely going to switch to it regardless because it has the close button, lol. Well, hopefully there will be some improvement/update made for it.

**In short, I'll take back the extras I've written 😂 and say my bad on this one for not remembering to check pass the addon I was already using.**

So, case closed (until the next one).

Thanks
